content/introduction-and-setup/walkthrough/mac-install/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Mac Users: Visual Studio for Mac"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Before you start the installation guide, read through each step below. You will find the recommended features and functionality to install listed in each step.
+
+{{% notice red "Warning" "rocket" %}}
+Allow at least **an hour** for the installation process.
+{{% /notice %}}
+
+## Installation Steps
+
+Walk through the installation process by following Microsoft’s [Install Visual Studio for Mac](https://learn.microsoft.com/en-us/visualstudio/mac/installation?view=vsmac-2022) guide.
+
+### Prerequisites
+
+Check out that you can install this on your computer.
+
+### Install Visual Studio
+
+1. Download the installer.
+1. Install using the `.dmg` file.
+1. Open the application.
+1. Continue if you agree with the privacy statements and license terms.
+1. The only component required for this course is **.NET**. You can modify this later as you gain more coding experience.
+1. Select the *Install* button.
+1. Assist the installer as needed. This may take a while, but stick around. You may be asked to enter your password a few times.
+1. To personalize or not? This is up to you.
+
+   1. Microsoft Account: Creating a Microsoft account is optional.
+   1. Keyboard Shortcuts: The default is Visual Studio for Mac and that is a good starting option. You can change this later.
+
+1. Launch Visual Studio and start developing
+
+## A Tour of Your New IDE
+
+Microsoft has created a [walkthrough](https://learn.microsoft.com/en-us/visualstudio/mac/ide-tour?view=vsmac-2022) of the Visual Studio 2022 for Mac. We recommend you read through this to learn the location of key components within your new IDE.
+
```

Comment:
Please add Visual Studio Extension information and Connect to GitHub for Mac users.

If there are some suggestions for Extensions that would be helpful.

Reply:
They don't need any extensions to complete this course, though I added some fun ones for both Mac and Windows users to the end of this chapter. Visual Studio also has Github functionality added in for both Windows and Mac users so the notes will be the same for both users so they are covered later.

Reply:
Originally the walkthrough for windows included information on extensions and github - the article was updated recently and no longer contains these references.